quadriga/src/main/java/edu/asu/diging/quadriga/core/mongo/impl/ElementDaoImpl.java

```diff
@@ -0,0 +1,26 @@
+package edu.asu.diging.quadriga.core.mongo.impl;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.quadriga.core.mongo.ElementDao;
+import edu.asu.diging.quadriga.domain.elements.Element;
+
+@Service
+public class ElementDaoImpl implements ElementDao {
+
+    @Autowired
+    private MongoTemplate mongoTemplate;
+
+    @Override
+    public void saveElements(List<List<Element>> elements) {
```

Comment:
saving should take one relationevent, which is the top element of the graph, or appellationevent.